de.metas.marketing/base/src/main/java/de/metas/marketing/base/model/ContactPersonRepository.java

```diff
@@ -93,44 +94,55 @@ private static I_MKTG_ContactPerson cerateOrUpdateRecordDontSave(
 	private static Optional<I_MKTG_ContactPerson> loadRecordIfPossible(
 			@NonNull final ContactPerson contactPerson)
 	{
+		final IQueryBL queryBL = Services.get(IQueryBL.class);
+
 		I_MKTG_ContactPerson contactPersonRecord = null;
+
 		if (contactPerson.getContactPersonId() != null)
 		{
 			final ContactPersonId contactPersonId = contactPerson.getContactPersonId();
 			contactPersonRecord = load(contactPersonId.getRepoId(), I_MKTG_ContactPerson.class);
```

Comment:
I believe that since we got the record,, here should be a return, right?

Reply:
Right. Fixed